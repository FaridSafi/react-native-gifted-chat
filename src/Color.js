export default {
  defaultColor: '#b2b2b2',
  backgroundTransparent: 'transparent',
  defaultBlue: '#0084ff',
  leftBubbleBackground: '#f0f0f0',
  white: '#fff',
  carrot: '#e67e22',
  emerald: '#2ecc71',
  peterRiver: '#3498db',
  wisteria: '#8e44ad',
  alizarin: '#e74c3c',
  turquoise: '#1abc9c',
  midnightBlue: '#2c3e50',
  optionTintColor: '#007AFF',
  timeTextColor: '#aaa',
};
