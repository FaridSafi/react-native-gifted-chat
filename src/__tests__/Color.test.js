import Color from '../Color';

it('should compare Color with snapshot', () => {
  expect(Color).toMatchSnapshot();
});
