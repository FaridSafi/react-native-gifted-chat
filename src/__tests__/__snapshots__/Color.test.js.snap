// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should compare Color with snapshot 1`] = `
Object {
  "alizarin": "#e74c3c",
  "backgroundTransparent": "transparent",
  "carrot": "#e67e22",
  "defaultBlue": "#0084ff",
  "defaultColor": "#b2b2b2",
  "emerald": "#2ecc71",
  "leftBubbleBackground": "#f0f0f0",
  "midnightBlue": "#2c3e50",
  "optionTintColor": "#007AFF",
  "peterRiver": "#3498db",
  "timeTextColor": "#aaa",
  "turquoise": "#1abc9c",
  "white": "#fff",
  "wisteria": "#8e44ad",
}
`;
